### ISSUE TITLE
Reset updateContext timeout is too long

### ISSUE THREAD

**Issue by onion2k** [TylerBarnes/gatsby-plugin-transition-link#22]
I've got a TransitionLink with exitLength, entryLength, and entryDelay all set to 5s. The transition works, but the call to reset the animation lengths to 0 and add the min-height to the div is taking a little too long - 15s rather than 10s.

I've had a look through the plugin code and _I think_ I've narrowed the problem down to line #78 of [triggerTransition](https://github.com/TylerBarnes/gatsby-plugin-transition-link/blob/3fe850fc863a8fa2cba95ba76821f6cfcb852cad/src/utils/triggerTransition.js#L78). It's doing "exitDelay + entryDelay + entryLength + exitLength", but as the entryDelay timeout runs concurrently with the exit animation according to the [timeline](https://transitionlink.tylerbarnes.ca/docs/timeline-of-events/) I think the calculation should be "exitDelay + entryLength + exitLength".

My TransitionLink code looks like this;

`<TransitionLink
    exit={{
        trigger: ()=>{},
        delay: 0,
        length: 5
    }}
    entry={{
        delay: 5,
        length: 5,
        trigger: ()=>{}
    }}
    to={'/'}
>`

**Comment by TylerBarnes** [comment]
Good catch! I think you're right. I'll get a fix in for this this weekend. Probably I'll check for which one is longer and use that one. Part of the reason for the calculation is to prevent people from clicking a link repeatedly during a longer transition. 
Thanks for noticing!
